luarules/gadgets/unit_respawning.lua

```diff
@@ -161,36 +180,39 @@ if gadgetHandler:IsSyncedCode() then
 			}
 
 			if respawnMetaList[unitID].effigy ~= "none" then
-				
-				for i = 1, 1000, 100 do
-					local x,y,z = spGetUnitPosition(unitID)
-					local blockType, blockID = Spring.GetGroundBlocked(x+i, z+i)
-					local groundH = Spring.GetGroundHeight(x+i, z+i)
-
+				local blockedIncriment = 1
+				for i = 1, 500, blockedIncriment do
```

Comment:
Is there a suitable function in the [position_checks.lua](https://github.com/beyond-all-reason/Beyond-All-Reason/blob/master/luarules/Utilities/damgam_lib/position_checks.lua) library that fulfills this functionality?

Reply:
Most likely. And I'll look through it and make the necessary change, but for now the current implementation works well enough to get it out there for live game testing.